contentwarehouse/snippets/quickstart_sample.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# [START contentwarehouse_quickstart]
+
+from google.cloud import contentwarehouse
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_number = 'YOUR_PROJECT_NUMBER'
+# location = 'YOUR_PROJECT_LOCATION' # Format is 'us' or 'eu'
+# schema_display_name = 'YOUR_SCHEMA_DISPLAY_NAME'
+# property_name = 'YOUR_PROPERTY_NAME' # Must be unique within a document schema (case insensitive)
+# property_display_name = 'YOUR_PROPERTY_DISPLAY_NAME'
+# property_is_searchable = True
+# document_display_name = 'YOUR_DOCUMENT_DISPLAY_NAME'
+# document_plain_text = 'YOUR_DOCUMENT_TEXT'
+# document_property_value = 'YOUR_PROPERTY_VALUE'
+
+
+def quickstart(
+    project_number: str,
+    location: str,
+    schema_display_name: str,
+    property_name: str,
+    property_display_name: str,
+    property_is_searchable: bool,
+    document_display_name: str,
+    document_plain_text: str,
+    document_property_value: str,
+) -> None:
+
+    # Create a Document schema
+    document_schema = create_document_schema(
+        project_number,
+        location,
+        schema_display_name,
+        property_name,
+        property_display_name,
+        property_is_searchable,
+    )
+
+    # Create a Document for the given schema
+    create_document_response = create_document(
+        project_number,
+        location,
+        document_schema,
+        document_display_name,
+        document_plain_text,
+        document_property_value,
+    )
+
+    # Read the output
+    print(f"Rule Engine Output: {create_document_response.rule_engine_output}")
+    print(f"Document Created: {create_document_response.document}")
+
+
+def create_document_schema(
```

Comment:
I would remove these extra functions (`create_document_schema` and `create_document_response`). Adding them increases the complexity and duplicates the (long) list of parameters passed to this sample.

The quickstart itself would be ~100 lines without them, which is fine for a quickstart. You'll be able to reduce the amount of duplicated code (instantiating the client, constructing the parent resource path).

Reply:
I did this to make it a little more clear about the steps involved, but if it is better without the extra functions, I'll change it

Reply:
Although, the Create Document Function uses a different client, so that code would still need to exist.
I think that's why I originally separated it out into functions. Each wraps a distinct function and client.